### DISCUSSION TITLE
Running in a subdirectory?

### DISCUSSION THREAD

**Discussion by mbrowne** [remix-run/remix#2281] [General]
Hi, is it possible to set a base URL so that a Remix app can be deployed in a subdirectory? Similar to next.js's `basePath` setting.

**Comment by kiliman** [comment]
I just created a new package that will let you mount your Remix app in a path other than root.

https://github.com/kiliman/remix-mount-routes

Example app
https://remix-mount-routes-example.herokuapp.com/

**Reply by mbrowne** [reply]
Awesome! Thanks for posting here!
